NuGet/IronSoftware.Drawing.nuspec

```diff
@@ -40,7 +40,8 @@ Supports:
 For general support and technical inquiries, please email us at: support@ironsoftware.com</description>
 		<summary>IronSoftware.System.Drawing is an open-source solution for .NET developers to replace System.Drawing.Common with a universal and flexible library.</summary>
 		<releaseNotes>- Efficient compression significantly reduces output TIFF file size (Up to 80%).
-- Automatic compression when creating multi-frame TIFFs.</releaseNotes>
+- Automatic compression when creating multi-frame TIFFs.
+- Fix the issue of exception error thrown when converting Bitmap to AnyBitmap.</releaseNotes>
```

Comment:
Replace with `Fixes the issue of exception being thrown when converting Bitmap to AnyBitmap.`